assignments/lambda-classes.js

```diff
@@ -1 +1,116 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor(name, age, location) {
+    this.name = name;
+    this.age = age;
+    this.location = location;
+  }
```

Comment:
Nice refactor to class component. You could also use 'data' or 'attr' instead of passing each individual argument.